src/clients/ntp-client_proposal.rb

```diff
@@ -203,13 +203,17 @@ def MakeProposal
       # Once read or proposed any config we consider it as read (bnc#427712)
       NtpClient.config_has_been_read = true
 
-      # do a proposal
-      ntp_sources = fallback_ntp_items
```

Comment:
Why are we modifying the defaults? In case no one is provided it should use the defaults ones from the control file isn't it? at least we are using them for the first sync.

Reply:
So, we modified it because the sources table was filled by the Country ntp servers but I guess we should use https://github.com/yast/yast-network/blob/9874173c8b9a15135da12621e4f50522cc3ec9dc/src/lib/y2network/ntp_server.rb#L58

Reply:
It is some magic which is done across the live of the dialog. 

First initialization is done in yast2-country, where sources from control files are read. However, later then in the ntp module some more logic is added. Like reading sources obtained from dhcp, "default hard coded" geo based servers. 

So, it simply is cumulation of hacks as different features got implemented over the time.

Reply:
I've modified this piece to make it work according to fate#323454) and bsc#1188980 hopefuly.